hoomd/mpcd/VirtualParticleFiller.cc

```diff
@@ -77,6 +54,20 @@ void mpcd::VirtualParticleFiller::setType(unsigned int type)
     m_type = type;
     }
 
+unsigned int mpcd::VirtualParticleFiller::computeFirstTag(unsigned int *N_fill)
+    {
+    unsigned int first_tag = 0;
+    #ifdef ENABLE_MPI
+    if (m_exec_conf->getNRanks() > 1)
+        {
+        // scan the number to fill to get the tag range I own
+        MPI_Exscan(N_fill, &first_tag, 1, MPI_UNSIGNED, MPI_SUM, m_exec_conf->getMPICommunicator());
```

Comment:
Then take the address here:
```suggestion
        MPI_Exscan(&N_fill, &first_tag, 1, MPI_UNSIGNED, MPI_SUM, m_exec_conf->getMPICommunicator());
```